pkg/storage/kubernetes/client.go

```diff
@@ -25,13 +26,16 @@ type Client struct {
 func NewClient(kubeconfigPath string) (*Client, error) {
 	scheme := runtime.NewScheme()
 	_ = whereaboutsv1alpha1.AddToScheme(scheme)
+	coordinationv1.AddToScheme(scheme)
 
 	config, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
 		&clientcmd.ClientConfigLoadingRules{ExplicitPath: kubeconfigPath},
 		&clientcmd.ConfigOverrides{}).ClientConfig()
 	if err != nil {
 		return nil, err
 	}
+	config.QPS = 50.0
+	config.Burst = 100
```

Comment:
client-go does rate limiting of requests towards api server and default qps/burst are 5/10.
https://pkg.go.dev/k8s.io/client-go/rest?utm_source=godoc#Config
Since, plugin ends up doing lots of requests towards api server, I changed the values.